File: Android/sukritoberoi_SukritKaurOberoi_2125cse1065_2/Patterns/Pattern2.java
import java.util.*;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        // write ur code here
        int n=sc.nextInt();
        for(int i=n;i>0;i--)
        {
            for(int j=0;j<i;j++)
            {
                System.out.print("*\t");
            }
            System.out.println();
        }

    }
}

File: AR-VR/Nandini-Maheshwari_NandiniMaheshwari_2125CSE1037_2/Patterns/Pattern2.cpp
#include <iostream>
using namespace std;

int main(){
    int n;
    cin >> n;
    
    //write your code here
    
    for(int i = n; i>0; i--){
        for(int j = i; j>0; j--){
            cout << "*\t"; 
        }
        cout << "\n";
    }
    return 0;
}